_posts/2016-08-09-collections-app.md

```diff
@@ -0,0 +1,82 @@
+---
+layout: post
+title: "Collecting Connections through the Powerbox"
+author: David Renshaw
+authorUrl: https://github.com/dwrensha
+---
+
+Starting work on a project in Sandstorm often means creating many grains, each pertaining to a
+different aspect of your work. In a typical project, you might have a spreadsheet, a chat room, a
+kanban board, and several source code repositories. Sandstorm makes it easy to share these grains
+with your collaborators, but until recently, you would need to share each grain to each collaborator
+separately --- a task that could quickly become tedious. What if you want to share all of the
+project's grains as a single unit?
+
+Now that we have released the
+[Collections app](https://apps.sandstorm.io/app/s3u2xgmqwznz2n3apf30sm3gw1d85y029enw5pymx734cnk5n78h),
+we have a satisfying answer to that question; to share many grains at once, you add them to a collection.
```

Comment:
I tested out the long-ish title of:

"Sandstorm supports collections, with the help of an app"

and it seems pretty reasonable to me:

![screenshot from 2016-08-09 17-09-54](https://cloud.githubusercontent.com/assets/25457/17533748/28dd9844-5e54-11e6-8e14-6590175044e5.png)

@xet7 Wow, thanks for replying! I'm glad my pinging you worked! Just to answer your questions above - yes, your understanding is accurate.